.github/workflows/tests-build-js.yml

```
@@ -71,15 +75,23 @@ jobs:
           echo ".idea" >> .gitignore
-          echo ".ultra.cache.json" >> .gitignore
+          echo ".ultra.exists.json" >> .gitignore
           echo "db/*" >> .gitignore
-        if: ${{ steps.cache.outputs.cache-hit != 'true' }}
+        if: ${{ steps.check-artifact.outputs.exists != 'true' }}
 
       - name: Get modified files
         id: diff
-        run: git ls-files --others --exclude-standard >> artifacts_list.txt
-        if: ${{ steps.cache.outputs.cache-hit != 'true' }}
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:24: Double quote to prevent globbing and word splitting

(shellcheck)